docs/notebook-demos/ingesting-an-image.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingesting an Image\n",
+    "\n",
+    "This example shows how to ingest a grayscale image and save the results locally. It is assumed that you have already read through the documentation on GeoPySpark before beginning this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the Data\n",
+    "\n",
+    "Before we can begin with the ingest, we must first download the data from S3. This curl command will download a file from S3 and save it to your `/tmp` direcotry. The file being downloaded comes from the [Shuttle Radar Topography Mission (SRTM)](https://www2.jpl.nasa.gov/srtm/index.html) dataset, and contains elevation data on the east coast of Sri Lanka.\n",
+    "\n",
+    "**A side note**: Files can be retrieved directly from S3 using the methods shown in this tutorial. However, this could not be done in this instance due to permission requirements needed to access the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!curl -o /tmp/cropped.tif https://s3.amazonaws.com/geopyspark-test/example-files/cropped.tif"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is an Ingest?\n",
+    "\n",
+    "Before continuing on, it would be best to discuss what an ingest actually is. In the context of GeoPySpark, it means to read in data and format it in such a way so that it can be analyzed and saved for later usage. Thus, `TiledRasterLayer` can be thought of as this final end stage."
```

Comment:
This is extremely vague.

_Before continuing on, it would be best to briefly discuss what an ingest actually is.  When data are acquired, they may cover an arbitrary spatial extent in an arbitrary projection.  These data need to be regularized to some expected layout and cut into tiles.  After this step, we will possess a `TiledRasterLayer` that can be analyzed and saved for later use._

Perhaps a reference to the layer guide would be useful here?